py/zk/zkns_query.py

```diff
@@ -41,7 +44,7 @@ def _sorted_by_srv_priority(entries):
     priority_map[entry.priority].append(entry)
 
   shuffled_entries = []
-  for priority, priority_entries in sorted(priority_map.iteritems()):
+  for unused_priority, priority_entries in sorted(priority_map.iteritems()):
```

Comment:
When I try to do a git pull, it says:

Already up-to-date.

When I do a checkout, it says:

Your branch is up-to-date with 'origin/master'.
The problem seems to be that even though everything is apparently
up-to-date, it is still making conflicts.
I think that it shouldn't be saying 'origin/master', it should be saying
'upstream/master'.


$ git remote -v
origin git@github.com:dumbunny/vitess.git (fetch)
origin git@github.com:dumbunny/vitess.git (push)
upstream git@github.com:youtube/vitess.git (fetch)
upstream git@github.com:youtube/vitess.git (push)

I'm going to try specifying upstream instead.

Dean Yasuda

On Sat, Aug 22, 2015 at 2:40 PM, Michael Berlin <notifications@github.com>
wrote:

> In py/zk/zkns_query.py
> <https://github.com/youtube/vitess/pull/1026#discussion_r37700057>:
>
> > @@ -41,7 +44,7 @@ def _sorted_by_srv_priority(entries):
> >      priority_map[entry.priority].append(entry)
> >
> >    shuffled_entries = []
> > -  for priority, priority_entries in sorted(priority_map.iteritems()):
> > +  for unused_priority, priority_entries in sorted(priority_map.iteritems()):
>
> You cannot use "_" here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/youtube/vitess/pull/1026/files#r37700057>.
>